Farm&Gun/Assets/Scripts/UI/LoaderInterface.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LoaderInterface : MonoBehaviour
```

Comment:
Maybe call this a little bit different, because its not an interface (c#-speaking) and Loader is a bit too vague, SceneLoaderAccessor perhaps, SceneLoaderSomething (change something)